src/components/TextField/index.tsx

```diff
@@ -0,0 +1,37 @@
+import classNames from 'classnames'
+import React from 'react'
+
+import classes from './TextField.module.scss'
+
+interface ITextField {
+  className?: string
+  id: string
+  name: string
+  label: string
+  type: string
+  value: string
+  onChange: (e: React.ChangeEvent<HTMLInputElement>) => void
+  error?: boolean | undefined | string
+  helperText?: string | undefined | boolean
+}
+
+const TextField = ({ className, id, name, label, type, value, onChange, error, helperText }: ITextField) => {
```

Comment:
I'd suggest to clarify the types for components as well.
```suggestion
const TextField: React.FC< ITextField> = ({ className, id, name, label, type, value, onChange, error, helperText }) => {
```

Reply:
Fixed